styles/tools/functions.test.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @jest-environment node
+ */
+
+import { join } from 'path';
+import { runSass } from 'sass-true';
+
+var sassFile = join(__dirname, 'functions_test.scss');
+console.log(sassFile);
```

Comment:
Let's use `const` instead of `var` here and delete the `console.log` statement. It isn't causing our linter to fail, but I'm getting squiggly lines in my editor.